lib/spack/spack/test/packaging.py

```diff
@@ -196,7 +196,7 @@ def test_relocate_text(tmpdir):
             script.close()
```

Comment:
To test the sbang relocation write a sbang line to the file here. Then you can pass the same sbang line to relocate_text.

Reply:
Nevermind.